apps/docs/src/app/platform/component-docs/platform-button/platform-button-examples/platform-button-types-example.component.html

```diff
@@ -1,7 +1,7 @@
-<fdp-button label="Standard Button"></fdp-button>
-<fdp-button buttonType="emphasized" label="Standard Button"></fdp-button>
-<fdp-button buttonType="ghost" label="Ghost Button"></fdp-button>
-<fdp-button buttonType="positive" label="Positive Button"></fdp-button>
-<fdp-button buttonType="attention" label="Attention Button"></fdp-button>
-<fdp-button buttonType="negative" label="Negative Button"></fdp-button>
-<fdp-button buttonType="transparent" label="Transparent Button"></fdp-button>
+<fdp-button ariaLabel="standard" label="Standard Button"></fdp-button>
+<fdp-button ariaLabel="standard" buttonType="emphasized" label="Standard Button"></fdp-button>
+<fdp-button buttonType="ghost" ariaLabel="ghost" label="Ghost Button"></fdp-button>
+<fdp-button buttonType="positive" ariaLabel="positive" label="Positive Button"></fdp-button>
+<fdp-button buttonType="attention" ariaLabel="attention" label="Attention Button"></fdp-button>
+<fdp-button buttonType="negative" ariaLabel="negative" label="Negative Button"></fdp-button>
+<fdp-button buttonType="transparent" ariaLabel="transparent" label="Transparent Button"></fdp-button>
```

Comment:
If it is necessary that the screenreader reads these buttons as "Standard Button" and not "Standard Button Button" etc then i think the inner text of the button should simply be "Standard" etc and no other changes would be required

Reply:
@mikerodonnell89 that might be one option, however that changes documentation, can we confirm this is a correct way to address it?

Reply:
1. Yes, you can simply change the label to `Standard` and this way the screen reader won't read Button twice
2. In my opinion it's not an issue at all and should not be addressed. This is concerning a specific example in OUR documentation. In a real application nobody will name their button `Save button` or `Submit button` :) We did because this is a documentation and does not represent a real case scenario.

Reply:
Hello Colleagues,

If you provide the button label as 'Standard Button' and the type is button.
Its not a bug.

Reply:
Hi @Sanjay-1312 , I don't think you read the whole conversation :) 
You still need to explain why we need aria label when there's text between the opening and closing tags of a button. Thanks